doc/WebSite/Data-Filters.md

```diff
@@ -37,19 +37,21 @@ done automatically by ASP.NET Boilerplate when you use the
 method (you can manually set IsDeleted to true, but the Delete method is the
 more natural and preferred way).
 
+In some cases, soft-delete entities may be requested to be permanently deleted. In those cases, **IRepository.HardDelete** extension method can be used. This method is currently implemented for EntityFramework 6.x and Entity Framework Core.
```

Comment:
This should be also document on https://aspnetboilerplate.com/Pages/Documents/Entities#soft-delete